test/new-e2e/utils/e2e/e2e.go

```diff
@@ -57,22 +56,19 @@ import (
 //
 // [testify Suite]: https://pkg.go.dev/github.com/stretchr/testify/suite
 type Suite[Env any] struct {
-	shortStackName string
-	fullStackName  string // From StackManager.Stack. Use shortStackName and the username
-	stackDef       *StackDefinition[Env]
-	destroyEnv     bool
+	suite.Suite
+
+	stackName string
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should we limit the length ?

Reply:
Why is `destroyEnv` gone ?

Reply:
I believe Pulumi will complain if the stack name is too long. `destroyEnv` is gone because I don't believe the setting belongs to a `TestSuite`, I believe it belongs to the running environment.